backend/scripts/request_runner.py

```
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+project_root = Path(__file__).resolve().parents[2]
+sys.path.append(str(project_root))
+from backend.config import GATEWAY_PORT
+
+
+def request_runner(query: str):
+    import requests
+    import json
+
+    response = requests.post(
```

Ruff (0.15.6)

[error] Probable use of `requests` call without timeout

(S113)